azure/iac/vm.rego

```diff
@@ -35,7 +34,7 @@ vm_aset_metadata := {
     "Language": "ARM template",
     "Policy Title": "Azure Virtual Machine is not assigned to an availability set",
```

Comment:
`Azure Virtual Machine should be assigned to an availability set`

Reply:
fixed